docs/0.1/getting-started/tutorial.md

```diff
@@ -1,7 +1,104 @@
 ---
 layout: docs
 title: Tutorial
-description: Coming Soon
+description: Getting Set-up in Microshare
 group: getting-started
 toc: true
 ---
+
+
+This tutorial will provide an simple review of Microshare. In this tutorial we will:
+
+1. Create an account  
+2. Get an access token  
+3. Post data to Microshare  
+4. Make a rule to govern this data  
+5. Create a robot to moniter this data  
+6. Make a dashboard to view this data  
+
+This process will require Postman. It is avaliable to download and install [here](https://www.getpostman.com/)
+
+## Post Data to Microshare
+
+Make a post call using the same parameters as before, but with two notable additions. 
+
+1. Include the data you wish to upload into the body of the call in JSON format. For example:
+
+    {"Test":"Data"}
+    
+2. Include a header with a key of "recType" 
+    * This will create a space for the data and attribute the above value as the recType's name  
+        * It is recommended that your first recType follow a simple naming convention I.E. "yourFirstName.yourLastName"
+
+
+## Make a Rule 
+
+A rule governs the data you have uploaded in a specific recType. You can adjust who can see the data and what they can do with it. In this section we will allow a user of your choosing to read and write to the data you have posted to microshare.
+
+1. Navigate to [our platform](../file)(https://dapp.microshare.io)
+2. Click the "Manage" button in the top toolbar
+3. Click the "Rules" button in the left toolbar
+4. Click the the "Create" button
+5. Input the Rule name in the top most text field
+5. Type "My First Rule" in the text field labeled "Description"
```

Comment:
Numbers here and below are stuck at 5

Reply:
It's ok - the numbers are like placeholders in markup. When viewed from the site it looks like a 1,2,3,4,5,6,7,8,9 - not just 5 over and over again.